api/constants.js

```diff
@@ -0,0 +1,5 @@
+const UNSUSCRIBETEXT = {
```

Comment:
Usually good practice to use snake case like `UNSUBSCRIBE_TEXT` to separate words and improve readability but obviously we're not enforcing this standard in other files so this is just an FYI